src/osm_configurator/model/project/project_io_handler.py

```diff
@@ -57,13 +44,39 @@ def convert_bool(string: str) -> bool | None:
     Returns:
         bool: The value of the string.
     """
-    if string == TRUE:
+    if string == saver_io_handler_constants.TRUE:
         return True
-    if string == FALSE:
+    if string == saver_io_handler_constants.FALSE:
         return False
     else:
         return None
 
+def _read_csv_file(path: Path) -> list[str]:
```

Comment:
those functions are not in a class. Is this fine? Because then we would have to take a lot of functions out of their classes

Reply:
I thought making such classes should be static, but if you want, i can revert it